programs/drift/src/math/amm.rs

```diff
@@ -171,18 +182,24 @@ pub fn estimate_best_bid_ask_price(
     // estimation of bid/ask by looking at execution premium
 
     // trade is a long
```

Comment:
if the trade is long, why wouldnt this be best_ask_estimate 🤔

Reply:
i moved that comment to be above relevant if/else hook

if taker is a long, hitting ask, assume best bid is `est_market_spread` below executed price